# DP: Revert the dwarf2cfi_asm workaround, obsoleted by PR debug/40521.

Revert:

2009-10-05  Ramana Radhakrishnan  <ramana.radhakrishnan@arm.com>

        * config/arm/arm.c (arm_override_options): Really initialize 
        flag_dwarf2_cfi_asm to 0.

Index: gcc/config/arm/arm.c
===================================================================
--- a/src/gcc/config/arm/arm.c	(revision 152450)
+++ b/src/gcc/config/arm/arm.c	(working copy)
@@ -1651,13 +1651,6 @@
         max_insns_skipped = 3;
     }
 
-  /* Ideally we would want to use CFI directives to generate
-     debug info.  However this also creates the .eh_frame
-     section, so disable them until GAS can handle
-     this properly.  See PR40521. */
-  if (TARGET_AAPCS_BASED)
-    flag_dwarf2_cfi_asm = 0;
-
   /* Register global variables with the garbage collector.  */
   arm_add_gc_roots ();
 }
